kube_helpers.go

```diff
@@ -110,7 +56,7 @@ func labelNamespace(ns string, labels map[string]string) {
 
 		exitCode, _, _ := cmd.exec(debug, verbose)
 		if exitCode != 0 && verbose {
-			log.Println("WARN: I could not label namespace [ " + ns + " with " + k + "=" + v +
+			logs.Warning("I could not label namespace [ " + ns + " with " + k + "=" + v +
```

Comment:
I would remove the "personnification" in the logs here to be consistent with log message refactoring. Maybe `namespace cannot be labeled [...]. It already exists. Skipping..`

Reply:
:+1: